src/activities/discussions/GroupEntity.js

```diff
@@ -0,0 +1,10 @@
+import { Entity } from '../../es6/Entity.js';
+
+export class GroupEntity extends Entity {
```

Comment:
I wonder if we should call this `NamedEntity`, since we're using it for both group and group-category, and it could potentially be used for other named entities that use the same format?